docs/sidebars.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       "behaviors/misc",
       "behaviors/hold-tap",
       "behaviors/mod-tap",
+      "behaviors/macros",
```

Comment:
Sticking macros right between mod-tap and mod-morph seems a bit odd. Should we maybe just alphabetize everything after the really common ones (key press, layers, misc.)?

Reply:
I tried to roughly do this, but splitting up hold- and mod-tap seemed wrong.

Reply:
At the very least I think we shouldn't split up mod-tap and mod-morph.

Reply:
Moved after mod-morph.